app/views/layouts/application.html.erb

```diff
@@ -10,12 +10,15 @@
     https://github.com/calebthompson/title#usage
   %>
   <title><%= title %></title>
+  <%= favicon_link_tag "bostonrb-logo.png" %>
   <%= stylesheet_link_tag :application, media: "all" %>
   <%= csrf_meta_tags %>
 </head>
 <body class="<%= body_class %>">
   <%= render "flashes" -%>
+  <%= render "layouts/header" %>
   <%= yield %>
+  <%= render "layouts/footer" %>
```

Comment:
Since this file is already in `layouts`, I think we can just call `render "footer"` here - same for `header` above.

Reply:
@mxie I had a similar thought process but referencing the footer and header partials without the `layouts` prefix results in the error: `Missing partial homes/_header, application/_header`. We could move them to the application directory but I prefer leaving them in `/layouts`. Thoughts?

Reply:
Oh, strange. Just leave it then. :)